Consul.Test/CoordinateTest.cs

```diff
@@ -49,5 +49,19 @@ public async Task Coordinate_GetNodes()
             // get an error. - from offical API.
             Assert.NotNull(nodes);
         }
+
+        [Fact]
+        public async Task Coordinate_GetNode()
+        {
```

Comment:
Previous version of the test was better, we did actually use a real node name in the query before.

Reply:
The thing is that with the node name I was receiving errors in the base fixture class